### ISSUE TITLE
Dockerfile does not install mmcv-full

### ISSUE THREAD

**Issue by namheegordonkim** [open-mmlab/mmdetection#3300]
See title. Current Dockerfile content is
```
ARG PYTORCH="1.5"
ARG CUDA="10.1"
ARG CUDNN="7"

FROM pytorch/pytorch:${PYTORCH}-cuda${CUDA}-cudnn${CUDNN}-devel

ENV TORCH_CUDA_ARCH_LIST="6.0 6.1 7.0+PTX"
ENV TORCH_NVCC_FLAGS="-Xfatbin -compress-all"
ENV CMAKE_PREFIX_PATH="$(dirname $(which conda))/../"

RUN apt-get update && apt-get install -y git ninja-build libglib2.0-0 libsm6 libxrender-dev libxext6 \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/*

# Install mmdetection
RUN conda clean --all
RUN git clone https://github.com/open-mmlab/mmdetection.git /opt/program
WORKDIR /opt/program
ENV FORCE_CUDA="1"
RUN pip install cython --no-cache-dir
RUN pip install "git+https://github.com/open-mmlab/cocoapi.git#subdirectory=pycocotools"
RUN pip install --no-cache-dir -e .
```

As per the latest installation instructions, this line:

```
RUN pip install mmcv-full --no-cache-dir
```

should be included somewhere before the last one.

**Comment by xvjiarui** [comment]
Hi @namheegordonkim 
Thanks for bringing this up. It will be fixed in #3320 .

**Comment by ganjbakhshali** [comment]
try these commands:
RUN git clone https://github.com/open-mmlab/mmcv.git 
WORKDIR mmcv 
RUN MMCV_WITH_OPS=1 pip install -e .
